src/components/DetailsOverlay/SearchRangeInput/index.tsx

```diff
@@ -0,0 +1,72 @@
+import styled from "@emotion/styled";
+import TextInput from "@leafygreen-ui/text-input";
+import { QueryParams } from "constants/queryParams";
+import { size } from "constants/tokens";
+import { useQueryParam } from "hooks/useQueryParam";
+import { FilterRow, StyledSubtitle } from "../styles";
+
+const SearchRangeInput: React.FC = () => {
+  const [lowerBound, setLowerQueryParam] = useQueryParam<number | undefined>(
+    QueryParams.LowerRange,
+    undefined
+  );
+  const [upperBound, setUpperBound] = useQueryParam<number | undefined>(
+    QueryParams.UpperRange,
+    undefined
+  );
+
+  const updateLowerBound = (newVal: string) => {
+    if (newVal === "") {
+      setLowerQueryParam(undefined);
+    } else {
+      setLowerQueryParam(parseInt(newVal, 10));
+    }
+  };
+
+  const updateUpperBound = (newVal: string) => {
+    if (newVal === "") {
+      setUpperBound(undefined);
+    } else {
+      setUpperBound(parseInt(newVal, 10));
+    }
+  };
+
+  return (
+    <FilterRow>
+      <StyledSubtitle> Range </StyledSubtitle>
+      <RangeContainer>
+        <RangeInput
+          aria-labelledby="Range Lower Bound"
```

Comment:
I think this should be aria-label.
 https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby

Reply:
The Leafygreen input will throw an error if it's `aria-label` with `type="number"`, it only accepts `aria-labelledby`